deploy/runtools/runtime_config.py

```diff
@@ -117,6 +123,22 @@ def get_local_driver_path(self) -> str:
         """ return relative local path of the driver used to run this sim. """
         return self.get_local_driver_dir() + self.get_local_driver_binaryname()
 
+    def tarball_creation_foldername(self) -> str:
+        """ return the name of the folder that the tarball is created from, no path """
+        return 'tarball'
+
+    def local_tarball_creation_path(self) -> str:
+        """ return the absolute path of the folder used to create the tarball """
+        triplet = self.get_deploytriplet_for_config()
+        dirname = self.tarball_creation_foldername()
+        return f'{get_deploy_dir()}/../sim/output/f1/{triplet}/{dirname}'
```

Comment:
By your estimation, how hard would it be to start using pathlib for these?

Reply:
I can try should be easy

Reply:
The f1 in this path needs to be the PLATFORM environment variable for this to work correctly under vitis.